EAN_chksum.py

```diff
@@ -0,0 +1,35 @@
+# EAN8/EAN13/SSCC/GTIN Code Verifier & Checksum Generator
```

Comment:
It seems this file is a seperated script instead of part of the servers? 
Please include some more comments to explain the usage of this file.